tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -90,8 +90,9 @@ before = (
 
 before_break1 = (
   <span>
-    <span barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar />
-    {" "}
+    <span
+      barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
+    />{" "}
```

Comment:
Hmm, this particular case looks like a regression to me... may not be a big deal.